pkg/domain/infra/tunnel/containers.go

```diff
@@ -162,6 +163,14 @@ func (ic *ContainerEngine) ContainerRm(ctx context.Context, namesOrIds []string,
 	var (
 		reports []*entities.RmReport
 	)
+	for _, cidFile := range options.CIDFiles {
+		content, err := ioutil.ReadFile(cidFile)
+		if err != nil {
+			return nil, errors.Wrap(err, "error reading CIDFile")
```

Comment:
Can you put the path of the CID file in the error?

Reply:
fixed